### PR TITLE
add 2 methods to Library

### PR DESCRIPTION
**Pull request by KatarinaSokhan**

### CODE REVIEW

**Comment by KatarinaSokhan** [comment]
LeSitar/TelRan_13M#6

**Comment by KatarinaSokhan** [comment]
HW6

File: 20200915_Library/src/de/telran/dao/Library.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            }
+
+        }
+        if (!flag) {
+            System.out.println("book is not found");
+        }
+    }
+
+    public Book getNewestBook() {
+        int lastYear = books[0].getDateOfIssue();
```
Comment (LeSitar) [inline 20200915_Library/src/de/telran/dao/Library.java:75]: redundant variable

File: 20200915_Library/src/de/telran/dao/Library.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        Book newestBook = books[0];
+        for (int i = 1; i < size; i++) {
+            if (books[i].getDateOfIssue() > lastYear) {
```
Comment (LeSitar) [inline 20200915_Library/src/de/telran/dao/Library.java:78]: instead of variable lastYear, you can use newestBook.getDateOfIssue

**Pull request merged**